github_connector/wizards/wizard_download_analyze_branch.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class WizardDownloadAnalyzeBranch(models.TransientModel):
+    _name = 'wizard.download.analyze.branch'
```

Comment:
I get this warning:
`wizard.download.analyze.branch has no _description`
so I think It's better that you add the parameter
`_description`